lib/testutils/httpmultibin/httpmultibin.go

```diff
@@ -115,6 +115,10 @@ func websocketEchoHandler(w http.ResponseWriter, req *http.Request) {
 		if err != nil {
 			break
 		}
+		err = conn.Close()
```

Comment:
I'm not sure I fully grok what's going on here, in this and the previous PR, but this `conn.Close()` doesn't make any sense to be in the `for` loop :confused: